qemu/tests/driver_in_use.py

```diff
@@ -0,0 +1,146 @@
+    1) boot guest with the device.
+    2) enable and check driver verifier in guest.
+    3) run subtest before / during / after background stress test.
+    4) clear the driver verifier.
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    def check_bg_running(session, target_process):
+        """
+        Check the backgroud test status in guest.
+
+        :param session: VM session.
+        :param target_process: Background process running in guest.
+        :return: return True if find the driver name;
+                 else return False
+        """
+        list_cmd = "wmic process where name='%s' list" % target_process
+        try:
+            output = session.cmd_output_safe(list_cmd, timeout=120)
+            check_reg = re.compile(r"%s" % target_process, re.I | re.M)
+            if bool(check_reg.findall(output)):
+                return True
+        except Exception, err:
+            raise exceptions.TestFail("Check process error: %s" % err)
+        return False
+
+    def bg_stress_test():
+        """
+        Run backgroud test.
+        """
+        bg_stress_test = params.get("run_bgstress")
+        if bg_stress_test:
+            try:
+                error_context.context("Run test %s background" % bg_stress_test,
+                                      logging.info)
+                stress_thread = ""
+                wait_time = float(params.get("wait_bg_time", 60))
+                target_process = params.get("target_process", "")
+                bg_stress_run_flag = params.get("bg_stress_run_flag")
+                env[bg_stress_run_flag] = False
+                stress_thread = utils.InterruptedThread(
+                    utils_test.run_virt_sub_test, (test, params, env),
+                    {"sub_type": bg_stress_test})
+                stress_thread.start()
+                if stress_thread.is_alive():
+                    try:
+                        stop_time = time.time() + wait_time
+                        while time.time() < stop_time:
+                            status = check_bg_running(session, target_process)
+                            if status:
+                                env["bg_status"] = 1
+                                break
+                    except Exception:
+                        raise exceptions.TestFail("Backgroud test %s is not"
+                                                  "alive!" % bg_stress_test)
+                if not utils_misc.wait_for(lambda: env.get(bg_stress_run_flag),
+                                           wait_time, 0, 5,
+                                           "Wait %s test start" % bg_stress_test):
+                    err = "Fail to start %s test" % bg_stress_test
+                    raise exceptions.TestError(err)
+                return stress_thread
+            except Exception:
+                raise exceptions.TestFail(err)
```

Comment:
Hi @xutian 

I have updated the code like the following code:
+            except Exception:
+                raise exceptions.TestFail("Background case %s start failed!" %
+                                          bg_stress_test)

I want to give some info to specify the background case failed.

What do you think?

Thanks.